roles/zabbix_agent/tasks/Windows.yml

```diff
@@ -191,8 +192,8 @@
 
 - name: "Windows | Set installation settings (agent 2)"
   ansible.builtin.set_fact:
-    zabbix_win_package: "{{ zabbix2_win_package }}"
-    zabbix_win_download_link: "{{ zabbix2_win_download_link }}"
+    zabbix_win_package: "{{ zabbix2_win_package | default('_win_package') }}"
+    zabbix_win_download_link: "{{ zabbix2_win_download_link | default('_win_download_link') }}"
```

Comment:
See the latest changes. Support for `zabbix2_win_download_link` has been removed with the release of 3.0.0